here-naksha-app-service/src/test/java/com/here/naksha/app/service/PropSearchTestUtil.java

```diff
@@ -0,0 +1,86 @@
+package com.here.naksha.app.service;
+
+import com.github.tomakehurst.wiremock.matching.RequestPatternBuilder;
+import org.apache.commons.lang3.tuple.Pair;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.util.stream.Stream;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.here.naksha.app.common.TestUtil.urlEncoded;
+
+public class PropSearchTestUtil {
```

Comment:
I like the idea of extracting samples of this amount to separate entity but I have two issues with this class:
1) Location. This class does not contain any tests - it's a sample provider, yet it lays right next to actual Test Suites. Consider moving it to `service.util` package
2) Name 😄  (nth & highly subjective) IMO this class is not a `Util`, it's just a bag for samples - functionality-wise it does not provide any mechanics. I would simply rename it to `PropertySearchSamples` ;)

Reply:
1. In service.util it still will be laying right next to actual Test Suites ;) but this time Test Suites unrelated to this class
2. Done